// swiper/modules/index.mjs
export {default as Virtual} from 'swiper/modules/virtual.mjs';
export {default as Keyboard} from 'swiper/modules/keyboard.mjs';
export {default as Mousewheel} from 'swiper/modules/mousewheel.mjs';
export {default as Navigation} from 'swiper/modules/navigation.mjs';
export {default as Pagination} from 'swiper/modules/pagination.mjs';
export {default as Scrollbar} from 'swiper/modules/scrollbar.mjs';
export {default as Parallax} from 'swiper/modules/parallax.mjs';
export {default as Zoom} from 'swiper/modules/zoom.mjs';
export {default as Controller} from 'swiper/modules/controller.mjs';
export {default as A11y} from 'swiper/modules/a11y.mjs';
export {default as History} from 'swiper/modules/history.mjs';
export {default as HashNavigation} from 'swiper/modules/hash-navigation.mjs';
export {default as Autoplay} from 'swiper/modules/autoplay.mjs';
export {default as Thumbs} from 'swiper/modules/thumbs.mjs';
export {default as FreeMode} from 'swiper/modules/free-mode.mjs';
export {default as Grid} from 'swiper/modules/grid.mjs';
export {default as Manipulation} from 'swiper/modules/manipulation.mjs';
export {default as EffectFade} from 'swiper/modules/effect-fade.mjs';
export {default as EffectCube} from 'swiper/modules/effect-cube.mjs';
export {default as EffectFlip} from 'swiper/modules/effect-flip.mjs';
export {default as EffectCoverflow} from 'swiper/modules/effect-coverflow.mjs';
export {default as EffectCreative} from 'swiper/modules/effect-creative.mjs';
export {default as EffectCards} from 'swiper/modules/effect-cards.mjs';
